lib/libutee/include/pta_system.h

```diff
@@ -13,6 +13,17 @@
 #define PTA_SYSTEM_UUID { 0x3a2f8978, 0x5dc0, 0x11e8, { \
 			 0x9c, 0x2d, 0xfa, 0x7a, 0xe0, 0x1b, 0xbe, 0xbc } }
 
+/*
+ * Having keys with too few bits impose a potential security risk, hence set a
+ * lower bound (i.e., 128 bits).
```

Comment:
maybe replace (i.e., 128 bits) with of 128 bits.

Reply:
You mean: `... hence set a lower bound of 128 bits`?

Reply:
yes

Reply:
Will fix.